src/ADAL.PCL/Utilities/Constants.cs

```diff
@@ -424,7 +424,7 @@ internal class XmlNamespace
 
         public static readonly XNamespace Soap12 = "http://schemas.xmlsoap.org/wsdl/soap12/";
         public static readonly XNamespace Wsa10 = "http://www.w3.org/2005/08/addressing";
-        public static readonly XNamespace Trust = "http://docs.oasis-open.org/ws-sx/ws-trust/200512";
+        public static XNamespace Trust = "http://docs.oasis-open.org/ws-sx/ws-trust/200512";
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Change the visibility of 'Trust' or make it 'const' or 'readonly'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://iddp-sonarqube:9000/coding_rules#rule_key=csharpsquid%3AS2223)